raiden/tests/integration/test_settlement.py

```diff
@@ -126,6 +130,101 @@ def test_settle_is_automatically_called(raiden_network, token_addresses, deposit
     })
 
 
+@pytest.mark.parametrize('number_of_nodes', [2])
+def test_lock_expiry(raiden_network, token_addresses, secret_registry_address, deposit):
```

Comment:
why is this a integration test?

the only thing interacting with the block chain is `waiting.wait_for_block`, which can be simulated with a `Block` state change.

Reply:
We agreed on having this as it exercises message encoding/decoding and transport.